src/en/proposals/mirrorcult-devil-redux.md

```diff
@@ -36,31 +37,47 @@ Making the devil not a crewmember, but just a separate antagonist, gives it more
 
 ## Contracts
 
-Beyond the worth expounding on the contract system, how it's intended to work, what type of gameplay it should foster, and possible counterplay. These are the bread and butter of devil gameplay.
+Beyond the worth expounding on the *unique* contract system, how it's intended to work, what type of gameplay it should foster, and possible counterplay. These are the bread and butter of devil gameplay.
 
-Each contract has a variable upside, one fixed downside, and a number of variable downsides. A devil can spawn a contract, but **anyone can edit a contract**, including the person signing it. The crewmember must sign it first, after which the devil can no longer edit the contract without voiding their signature. When the devil signs the contract, it goes into effect.
+Contracts are **simply fancy pieces of paper** which can be written on. However, they do have some fixed areas--a header, bullet point list, and footer. Since it's just text, you can use font changes/size changes, color changes, etc (to a certain degree).
 
-Upsides are things such as:
+A devil can spawn and create a contract, but **anyone can edit a contract**, including the person signing it. The crewmember must sign it first, after which the devil can no longer edit the contract without voiding their signature. When the devil signs the contract, it goes into effect.
+
+Each contract has a fixed downside that, when the crewmember signs it, **they sign away their soul** and will **go to hell when they die** (not able to be revived, forced into a hell dimension).
+
+Each contract has a number of **clauses** which are listed in bullet points. Clauses are **written in natural language** and use an extremely rudimentary parsing system to determine their effect. Each clause has a limited number of characters, and they have three different ways of changing their behavior: *triggers*, *positive/negative effects*, and *modifiers*.
+
+**Triggers** are how the clause will be triggered. A clause may have as many triggers as you can fit. Normally, it's simply "on signing the contract", however some other triggers are available:
+- "after X minutes/seconds", simply delaying the
```

Comment:
delaying the

Reply:
the

Reply:
t


















he

Reply:
Me when